Assets/Scripts/GlyphRecognition.cs

```diff
@@ -93,6 +111,7 @@ bool IsClear(StrokeGraphic strokeGraphic)
 		// Debug.Log(match.Cost);
 		switch (match.target.ToString()) {
 			case "FireGlyph":
+			case "FireBallGlyph":
```

Comment:
Not really all that important but we probably don't need to add Glyph to the end of all these names